src/crud.js

```diff
@@ -208,9 +250,13 @@ export const update = (server, model) => {
       validate: {
         payload: joi.object().required(),
       },
-    }, defaultConfig),
+    }, config),
   });
 };
 
 import * as associations from './associations/index';
 export { associations };
```

Comment:
Can do. They were here before and I thought it was weird, but wanted to keep the diff minimal.